src/NuGet/BuildNuGets.csx

```diff
@@ -212,8 +213,27 @@ int PackFiles(string[] nuspecFiles, string licenseUrl)
 
         if (!IsCoreBuild)
         {
+            string packageArgs = commonArgs;
+
+            // HACK: since Microsoft.Net.Compilers 2.0.0 was uploaded by accident and later deleted, we must bump the minor.
+            // We will do this to both the regular Microsoft.Net.Compilers package and also the netcore package to keep them
+            // in sync.
+            if (BuildVersion.StartsWith("2.0.") && Path.GetFileName(file).StartsWith("Microsoft.Net.Compilers."))
+            {
+                string[] buildVersionParts = BuildVersion.Split('-');
+                string[] buildVersionBaseParts = buildVersionParts[0].Split('.');
+                
+                buildVersionBaseParts[buildVersionBaseParts.Length - 1] =
+                    (int.Parse(buildVersionBaseParts[buildVersionBaseParts.Length - 1]) + 1).ToString();
```

Comment:
Don't see how I'd fail in any way other than throwing an exception anyways. (No, I'm not logging an error and continuing. Errors should be fatal here.)